src/markdown-pages/docs/managers/github-manager.md

```diff
@@ -0,0 +1,290 @@
+
+If you are using Maven, add the following dependencies into the _pom.xml_ in the _dependencies_ section, editing the version number to be the correct number for your configuration:
+
+```
+<dependency>
+<groupId>dev.galasa</groupId>
+<artifactId>dev.galasa.githubissue.manager</artifactId>
+<version>0.25.0</version>
+</dependency>
+```
+
+If you are using Gradle, add add the following dependencies into ```build.gradle``` in the _dependencies_ section, editing the version number to be the correct number for your configuration:
+
+```
+compileOnly 'dev.galasa:dev.galasa.githubissue.manager:0.25.0'
+```
+
+# <a name="annotations"></a>Annotations provided by the Manager
+
+The following annotation is provided by the GitHub Manager:
+
+<details>
+<summary>GitHub Issue</summary>
+
+| Name: | GitHub Issue |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @GitHubIssue |
+| Description: | If present on a method or class, this annotation will tell the GitHub Manager to influence the result of the method or class based on the known problem in the GitHub issue. |
+| Syntax:  | @GitHubIssue( githubId = "DEFAULT", issue = "1000", repo = "galasa-dev/projectmanagement")|
+| Attribute `issue`: | The number of the issue. Required.|
+| Attribute `githubId`: | Optional. Default value is "DEFAULT".|
+| Attribute `repo`: | Optional. If not present in the annotation, the CPS is checked. If a value is not present in the CPS, the Manager logs a warning message.|
+| Attribute `regex`: | Optional. Use to narrow down the failing exception.|
+
+The annotation can be used at the class or method level.
+
+Use of the _@ContinueOnTestFailure_ annotation alongside the _@GitHubIssue_ annotation can affect the test result. For examples, see the [Overrides](#overrides) section.
+
+</details>
+
+
+# <a name="codesnippets"></a>Code snippets and examples
+
+<details><summary>Add the @GitHubIssue and @ContinueOnTestFailure annotation on a test class</summary>
+
+Use the following code to add GitHub issue _1000_ in the _galasa-dev/projectmanagement_ repository on the SimBank _BatchAccountsOpenTest_ test class. Ensure all test methods in the test class are run by using the _@ContinueOnTestFailure_ annotation:
+
+```
+@Test
+@GitHubIssue( issue = "1000", repo = "galasa-dev/projectmanagement" )
+@ContinueOnTestFailure
+public class BatchAccountsOpenTest {
+```
+</details>
+
+<details><summary>Add the @GitHubIssue annotation on a test method</summary>
+
+Use the following code to add GitHub issue _1000_ in the _galasa-dev/projectmanagement_ repository on the _testNotNull_ method in the _BatchAccountsOpenTest_ test class:
+
+```
+@Test
+@GitHubIssue( issue = "1000", repo = "galasa-dev/projectmanagement" )
+public void testNotNull() {
+    // Check all objects loaded
+    assertThat(terminal).isNotNull();
+    assertThat(resources).isNotNull();
+    assertThat(client).isNotNull();
+}
+```
+</details>
```

Comment:
Since we've used this before why do we need to mention it again?

Reply:
updated the examples to use a different test